builtin/credential/approle/path_role.go

```diff
@@ -948,16 +1021,17 @@ func (b *backend) pathRoleRead(ctx context.Context, req *logical.Request, data *
 	}
 
 	respData := map[string]interface{}{
-		"bind_secret_id":     role.BindSecretID,
-		"bound_cidr_list":    role.BoundCIDRList,
-		"period":             role.Period / time.Second,
-		"policies":           role.Policies,
-		"secret_id_num_uses": role.SecretIDNumUses,
-		"secret_id_ttl":      role.SecretIDTTL / time.Second,
-		"token_max_ttl":      role.TokenMaxTTL / time.Second,
-		"token_num_uses":     role.TokenNumUses,
-		"token_ttl":          role.TokenTTL / time.Second,
-		"local_secret_ids":   false,
+		"bind_secret_id":        role.BindSecretID,
+		"secret_id_bound_cidrs": role.SecretIDBoundCIDRs,
+		"token_bound_cidrs":     role.TokenBoundCIDRs,
```

Comment:
For discussion: replacing `bound_cidr_list` here with the two new parameters could break people's code. Should I return them all?

Reply:
Yeah, it will break backwards compatibility. I am leaning towards replicating `secret_id_bound_cidrs` in another field `bound_cidr_list` with an additional warning added to the response object that `bound_cidr_list` is deprecated and will not be returned in future releases. (and a note in code as a reminder for us to remove it).